src/SonarQube.Client/Models/Plugins.cs

```diff
@@ -18,18 +18,33 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Diagnostics;
 using Newtonsoft.Json;
 
-namespace SonarLint.VisualStudio.Integration.Service.DataModel
+namespace SonarQube.Client.Models
 {
-    [DebuggerDisplay("Key: {Key}, Version: {Version}")]
-    internal class ServerPlugin
+    public class SonarQubePlugin
+    {
+        public string Key { get; }
+        public string Version { get; }
+
+        public SonarQubePlugin(string key, string version)
+        {
+            Key = key;
+            Version = version;
+        }
+
+        public static SonarQubePlugin FromDto(PluginDTO dto)
+        {
+            return new SonarQubePlugin(dto.Key, dto.Version);
+        }
+    }
+
+    public class PluginDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'PluginDTO' to match camel case naming rules, consider using 'PluginDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)